wasm/src/project_system.rs

```diff
@@ -165,14 +166,34 @@ pub(crate) fn get_manifest_transformer(js_val: JsValue, _: String) -> Option<Man
                 v
             )
         }),
-                    Err(_) => unreachable!("our typescript bindings should guarantee that an object with a manifestDirectory property is returned here"),
+        Err(_) => unreachable!("our typescript bindings should guarantee that an object with a manifestDirectory property is returned here"),
```

Comment:
I feel like most of this function could be replaced with `serde_wasm_bindgen::from_value` as long as you define a `ManifestDescriptor` struct with the proper fields and marked as `Deserializable`.

Reply:
Oops, I didn't realize this was just a whitespace change. I still think `serde_wasm_bindgen::from_value` should be directly callable, I'll try to sketch up an example, but don't worry about holding up the PR for that.